test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Roundtrip/JsonReaderWriterInjectionTests.cs

```diff
@@ -139,6 +322,38 @@ private static ODataResource GetReadedResource(string messageContent, EdmModel m
             }
         }
 
+        private static IList<ODataResource> GetReadedResourceWithNestedInfo(string messageContent, EdmModel model, IEdmEntitySetBase entitySet, EdmEntityType entityType, IServiceProvider container)

```

Comment:
>container [](start = 187, length = 9)

where do you use the container?

Reply:
Inside the method. (copied from existing code)
 IODataResponseMessage message = new InMemoryMessage
            {
                Stream = outputStream,
                Container = container
            };